pythonx/UltiSnips/vim_helper.py

```diff
@@ -218,10 +218,15 @@ def get_dot_vim():
     candidates = []
     if platform.system() == "Windows":
         candidates.append(os.path.join(home, "vimfiles"))
+
     if vim.eval("has('nvim')") == "1":
```

Comment:
```suggestion
    if vim.eval("has('nvim')") == "1" or vim.eval("$XDG_CONFIG_HOME"):
```

and delete the rest of the code?

Reply:
We would need to append both paths in that case, but I realized this is the case when having nvim with the current code.

```
    if vim.eval("has('nvim')") == "1"  or vim.eval("$XDG_CONFIG_HOME"):
        xdg_home_config = vim.eval("$XDG_CONFIG_HOME") or os.path.join(home, ".config")
        candidates.append(os.path.join(xdg_home_config, "nvim"))
        candidates.append(os.path.join(xdg_home_config, "vim"))
```